src/screens/Home/index.tsx

```diff
@@ -6,6 +6,8 @@ const HomeScreen = (): JSX.Element => {
   return (
     <div className="app">
       <header className="app-header">
+        {/* TODO: This is the test line for .env file. Will remove it later. */}
+        <div>{process.env.REACT_APP_DUMMY_ENV}</div>
```

Comment:
How did you save this variable in Netlify?
Did you save it as `DUMMY_ENV` or `REACT_APP_DUMMY_ENV`?
Reading from the SO answer you shared, it seems only `REACT_APP_DUMMY_ENV`  will work!

Also if it doesn't work, could you try out another action:
https://github.com/marketplace/actions/netlify-actions

Reply:
@tyrro I am using the action you shared 😄 and I saved the env as `REACT_APP_...` because React only recognize the env var with `REACT_APP_` prefix. Due to [this problem](https://nimble-co.slack.com/archives/C05C7PJ25FA/p1687161957618329), I cannot save the env on Netlify, instead I save them in `ENV_FILE_CONTENT` var of Github action 🙏